test/source/tests/gmail.ts

```diff
@@ -188,14 +189,16 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
     }));
 
     ava.default('mail.google.com - rendering attachmnents', testWithBrowser('ci.tests.gmail', async (t, browser) => {
-      const gmailPage = await openGmailPage(t, browser, '/FMfcgzGkbDXBWBWBfVKHssLtMqvDQSWN');
```

Comment:
For some weird reason this stopped working, it opens the list of emails instead of particular thread.

The fix is to open inbox, then wait 0.5s, then open a thread.